cameraview/src/main/java/com/otaliastudios/cameraview/CameraListener.java

```diff
@@ -127,4 +127,16 @@ public void onZoomChanged(float newValue, @NonNull float[] bounds, @Nullable Poi
     @UiThread
     public void onExposureCorrectionChanged(float newValue, @NonNull float[] bounds, @Nullable PointF[] fingers) { }
 
+
+    /**
+     * Notifies that the actual video recording has started
+     * When we request for video recording, it doesn't start the recording the immediately.
+     * Instead it checks for hardware availability first and later on available,
+     * it sets all the config and then start the actual recording.
```

Comment:
Instead of explaining the internals can you specify that from this moment, the engine will start recording frames into the final video, and this callback can be used, for example, to show a "recording" indicator?